inc/composer/class-docker-compose-generator.php

```diff
@@ -237,6 +237,53 @@ protected function get_service_php() : array {
 		];
 	}
 
+	/**
+	 * Get the NodeJS container service.
+	 *
+	 * @return array
+	 */
+	protected function get_service_nodejs() : array {
+		$config = $this->get_config();
+
+		$version = (string) $this->get_config()['nodejs']['version'] ?? '20';
+
+		return [
+			'nodejs' => [
+				'image' => "node:{$version}-bookworm",
+				'container_name' => "{$this->project_name}-nodejs",
+				'ports' => [
+					'3000',
+				],
+				'volumes' => [
+					"../{$config['nodejs']['path']}/:/usr/src/app"
+				],
+				'working_dir' => '/usr/src/app',
+				'command' => 'npm run dev',
+				'networks' => [
+					'proxy',
+					'default',
+				],
+				'healthcheck' => [
```

Comment:
I'm not sure about an HTTP healthcheck. if my node add has a fatal, I don't really want my container to start restarting? Or maybe I do? I don't know, can you explain the usecase for the healthcheck

Reply:
I did not consider those. I just copied how it was implemented on other services we run. Your point makes sense though, since the container will continually watch for file changes and recompile. 

I'll remove it for now.